opcode.c

```diff
@@ -35,6 +35,24 @@ static void mul_impl(VM_HANDLER_ARGS)
     }
 }
 
+static void div_impl(VM_HANDLER_ARGS)
+{
+    if (VM_T(op1) == INT && VM_T(op1) == VM_T(op2)) {
+        VM_INT(result) = VM_INT(op1) / VM_INT(op2);
```

Comment:
Maybe here can insert a line to check `op2`'s value, e.g. `if (VM_INT(op2) == 0`.

Also, will it need to convert the type? `(size_t) (VM_INT(op1) / VM_INT(op2))